store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedStoreToFederatedStoreTest.java

```diff
@@ -61,61 +62,64 @@
     @BeforeEach
     public void setUpStores() throws OperationException {
         SingleUseFederatedStore.cleanUp();
+        CacheServiceLoader.shutdown();
 
         ProxyProperties proxyProperties = new ProxyProperties();
         proxyProperties.setStoreClass(SingleUseFederatedStore.class);
 
         restApiFederatedGraph = new Graph.Builder()
-            .storeProperties(proxyProperties)
-            .config(new GraphConfig("RestApiGraph"))
-            .addSchema(new Schema())
-            .build();
+                .storeProperties(proxyProperties)
+                .config(new GraphConfig("RestApiGraph"))
+                .addSchema(new Schema())
+                .build();
 
+        FederatedStoreProperties properties = new FederatedStoreProperties();
+        properties.setCacheServiceNameSuffix("Alt");
```

Comment:
Is this just to make it easier in debugging?

Reply:
Again so long I'm unsure, It might have been to identify the Cache is not Default.